decoder.go

```diff
@@ -101,8 +103,13 @@ func (dec *Decoder) Decode() (Card, error) {
 	return card, nil
 }
 
-func parseLine(l string) (key string, field *Field, err error) {
-	field = new(Field)
+const placeholder = "\uFFFD"
```

Comment:
Instead of using a placeholder (which could be in the input), can we replace this logic with a proper char-by-char parser?